doc/manual/manual.tex

```diff
@@ -10432,6 +10432,89 @@ \subsubsection{Donea \& Huerta 2D box geometry benchmark}
 
 
 
+\subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conversion benchmark}
+\label{sec:benchmark-yamauchi-takei}
+
+\textit{This section was contributed by Fred Richards.}
+
+This benchmark tests the implementation of the anelastic shear wave velocity-to-temperature conversion derived by Yamauchi \& Takei~\cite{YT16} based on forced-oscillation experiments conducted at seismic frequencies on polycrystalline borneol. This anelasticity parameterization has been calibrated against a range of observational constraints on upper mantle temperature, attenuation and viscosity structure, using the surface wave tomography model of Priestley et al.\cite{P12} to constrain shear wave velocity (V$_S$) and the plate model of \cite{McK05} to estimate lithospheric temperature structure. The resulting V$_S$-to-temperature conversion accurately accounts for the strongly non-linear temperature dependence of V$_S$ at near-solidus conditions and is therefore especially useful for initializing models with accurate temperature structure in the upper $\sim$~400 km of the mantle.
+
+\begin{figure}
+\begin{center}
+  \centering
+  \includegraphics[width=\textwidth]{cookbooks/benchmarks/anelasticity/YT16_benchmark.jpg}
```

Comment:
did you put the .jpg files in the correct place? (doc/manual/cookbooks/benchmarks/...)

Reply:
these are in the correct place as far as I can tell (compiled correctly on my machine), however,  I have changed the file to a .png to make the file size a bit smaller.